include/net/net_stats.h

```diff
@@ -222,6 +230,7 @@ struct net_stats_tc {
 	} sent[NET_TC_TX_COUNT];
 
 	struct {
+		struct net_stats_rx_time rx_time;
```

Comment:
But this means that this is cumulative statistics, right? Nowhere does commit message makes this clear. The way it's worded, it seems that you collect per-packet stats.